journal_name.py

```diff
@@ -3,33 +3,50 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import json
+import pdb
+import types
 
+def store(data,file_name):
+    with open(file_name, 'w') as json_file:
+        json_file.write(json.dumps(data))
+
+        
 def get_json():
 	jour=0
 	conf=0
 	list=[]
 	diction={}
 	number=[]
 	name=[]
-	with open('bibtex1.json') as json_file:
-		data = json.load(json_file)
+	bibdic={}
+	n=0
+	with open('bibtex.json') as json_file:
+		data = json.load(json_file) #type of data is list
 		for a in data:
-			b = a.split('\n')
-			for m in b: 
-				if "journal={" in m:
+			c=a.split("\n")  #type of data is list
```

Comment:
Don't split by "\n". Your input looks like this:

```
@article{hardin2014clinical, title={The clinical and genetic features of COPD-asthma overlap syndrome}, 
author={Hardin, Megan and Cho, Michael and McDonald, Merry-Lynn and Beaty, Terri and Ramsdell, 
Joe and Bhatt, Surya and Van Beek, Edwin JR and Make, Barry J and Crapo, James D and Silverman, 
Edwin K and others}, journal={European Respiratory Journal}, volume={44}, number={2}, pages={341--350}, 
year={2014}, publisher={Eur Respiratory Soc} } "
```

Text between `@` and `{` is the article type. Next take everything inside `{ }` and split by comma, remove the first item from the result, then split each item by `=`, the first item in the result will be the attribute name, the second item, after removing `{ }` will be the attribute value.